### ISSUE TITLE
How viable to use this out in the wild?

### ISSUE THREAD

**Issue by nukepuppy** [flosell/iamspec#2]
How viable to use this out in the wild?

Just thinking i noticed this is a WIP/POC but what else could be done to make this more useable?

**Comment by flosell** [comment]
I left the project that inspired me to build this before I had a chance to really gain experience using it in practice on a larger scale. Also, my focus lay elsewhere in the last few months so I wasn't prepared to invest much time but I still wanted to put it out there, see what people think - that's where the WIP/POC notes come from. 

The [TODO section of the Readme](https://github.com/flosell/iamspec/blob/3cf4b9a/README.md#todo) mentions a few things I felt needed to be done. The biggest one there is probably syntactic sugar: Some cases might be cumbersome to express in the current state and only practice will tell what those are. I'll always be happy for PRs and open to making people contributors if they find the project useful and want to take a more active role shaping it. 

Long story short, if this is project is helpful to you, there's nothing that's stopping you from using it. Just don't expect it to be a finished, stable, actively maintained project at this point.

**Comment by nukepuppy** [comment]
@flosell  thanks! ill have to evaluate that ;) thanks either way
